.github/workflows/update-solana.yaml

```diff
@@ -0,0 +1,25 @@
+name: Update Solana
+
+on:
+  schedule: 
+    - cron: "00 00 * * *" # Run this workflow daily
+  workflow_dispatch:
+
+jobs:
+  update-solana:
+    name: Update Solana
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - uses: clockwork-xyz/update-solana@v1.2.0
+      - uses: peter-evans/create-pull-request@v4
+        with:
+          title: Update Solana to v${{ env.SOLANA_VERSION }}
+          base: master
+          branch: update-solana/stable
+          commit-message: Update "solana-" dependencies to v${{ env.SOLANA_VERSION }}
+          draft: false
+          delete-branch: true
+          labels: dependencies
+          reviewers: nickgarfield
```

Comment:
@Henry-E Who should be set as a reviewer on these auto-generated PRs?

Reply:
Me but like we discussed I'm not quite motivated to include this yet.